src/content/web3/index.md

```diff
@@ -0,0 +1,148 @@
+- **Web3 is permissionless:** everyone has equal access to participate in web3, and no one gets excluded.
+- **Web3 has native payments:** it uses cryptocurrency for spending and sending money online instead of relying on the outdated infrastructure of banks and payment processors.
+- **Web3 is trustless:** it operates using incentives and economic mechanisms instead of relying on trusted third-parties
+
+### Why is web 3.0 important? {#why-is-web3-important}
+
+So why does web3 matter? Web3's killer features aren't isolated and don't fit into neat categories. But, for simplicity, we've tried to separate them to make them easier to understand.
+
+#### Ownership {#ownership}
+
+Web3 gives you ownership of your digital assets in an unprecedented way. For example, say you're playing a web2 game. If you purchase an in-game item, it is tied directly to your account. If the game creators delete your account, you will lose these items. Or, if you stop playing the game, you lose the value you invested into your in-game items.
+
+Web3 allows for direct ownership through [non-fungible tokens (NFTs)](/nft/). No one, not even the game's creators, has the power to take away your ownership. And, if you stop playing, you can sell or trade your in-game items on open markets and recoup their value.
+
+<ButtonLink to="/nft/">
+  More on NFTs
+</ButtonLink>
+
+#### Governance {#governance}
+
+As a Web2 platform increases in popularity, the power of the individual user's opinion decreases. Simultaneously, the power of the platform's governing body also increases. Over time, this power imbalance can spiral out of control until the platform can ignore grievances with no repercussions.
+
+Web3 empowers users to participate in the decision making of a platform through tokens. A recent example of this is Ethereum Name Service (ENS), which distributed 25% of its voting rights to existing users by sending ENS tokens to their wallets. Anyone with ENS tokens can vote on proposals about the future of ENS. ENS is a decentralized autonomous organization (DAO) because its ownership gets distributed amongst its token holders.
+
+_In Web2, you can refuse to use a platform; in Web3, you can vote to change it._
+
+<ButtonLink to="/dao/">
+  More on DAOs
+</ButtonLink>
+
+#### Censorship resistance {#censorship-resistance}
+
+The power dynamic between platforms and content creators is massively imbalanced.
+
+OnlyFans is a user-generated adult content site with over 1-million content creators, many of which use the platform as their primary source of income. In August 2021, OnlyFans announced plans to ban sexually explicit content. The announcement sparked outrage amongst creators on the platform, who felt they were getting robbed of an income on a platform they helped create. After the backlash, the decision got quickly reversed.
+
+Web 2.0 requires content creators to trust platforms not to change the rules, but censorship resistance is a native feature of a web3 platform.
```

Comment:
Right, I would agree with this comment. This is kind of an example of how OnlyFans _could have_ banned if they wanted to. A social graph build on decentralized rails could allow creators to port their content to another interface for example, or have it served on many front-ends, so if one decides to filter out content, users can just use another one where the same creator is listed.

Reply:
The idea was to show a tangible example of the power held by Web2 companies to censor users. The fact that it was flouted wasn't meant to be important. 

Overall I like this example because it's popular, relatable, and apolitical. Taking into account we want this to be approachable to newbies I think we should start with some sort of tangible example of censorship.

Do we think expanding this example to talk how despite this attempt at censorship failing due to users threatening to vote with their feet, they would have lost all of their data/reputation/followers otherwise and then discussing how open permissionless code fixes this?

Reply:
I think there are better examples where censorship succeeded (like YouTube, Facebook, Twitter, etc.).  Using an example where the censorship failed to censor people makes for a poor example for why we need censorship resistant systems.

For any successful censorship, there will almost always be some people who agree with it, which is why it is able to stick around.  This means that successful censorship examples will essentially always be political.  It is only "apolitical" in the cases where the censorship fails (because everyone agrees it is bad).

Reply:
I've created a ticket to iterate on this. Merging this PR for now and will rethink this example in the first iteration.